Gemfile

```diff
@@ -2,14 +2,6 @@
 
 source 'https://rubygems.org'
 
-if RUBY_VERSION.start_with? '1.9'
-  gem 'net-ldap', '< 0.13'
-end
```

Comment:
This part triggered a security warning on Github. We don't really care about it, but this PR formalizes the dropping of old versions.